test/new.jl

```diff
@@ -284,7 +284,7 @@ end
 
 @testset "test: fallback when no project file exists" begin
     isolate(loaded_depot=true) do
-        Pkg.add(name="Permutations", version="0.3.2")
+        Pkg.add(name="Permutations", version="0.4.9")
```

Comment:
But now this version have a project file and this test is useless.

Reply:
Ooof. Brutal.

Should we make a dummy example package ourselves, and use that for the test?

Reply:
Why not just guard the test here based on the arch instead?

Reply:
Sure, we can do that instead.

Reply:
Done.